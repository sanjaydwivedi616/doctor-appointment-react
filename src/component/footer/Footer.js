import React from 'react'

function Footer() {
    return (
        <p className="footer">&#169; copyright HCL 2021</p>
    )
}

export default Footer;