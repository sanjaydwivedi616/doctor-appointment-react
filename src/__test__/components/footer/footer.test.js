import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../../component/footer/Footer';

it('Footer text is render', () => {
  const footerText = renderer.create(<Footer></Footer>)
  expect(footerText).toMatchSnapshot();
});