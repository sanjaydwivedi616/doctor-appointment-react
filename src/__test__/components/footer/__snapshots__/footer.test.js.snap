// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer text is render 1`] = `
<p
  className="footer"
>
  Â© copyright HCL 2021
</p>
`;
